packages/types/src/indexed-db.ts

```diff
@@ -122,9 +123,9 @@ export interface PenumbraDb extends DBSchema {
     key: 'params';
     value: Jsonified<FmdParameters>;
   };
-  TRANSACTION_INFO: {
-    key: Jsonified<Required<TransactionInfo>['id']['inner']>; // base64
-    value: Jsonified<TransactionInfo>;
+  TRANSACTIONS: {
+    key: string; // base64 TransactionInfo['id']['inner'];
+    value: Jsonified<TransactionInfo>; // TransactionInfo with undefined view and perspective
```

Comment:
I was originally thinking of using a custom type to save transactions, something like 
```typescript
export interface TransactionRecord {
  id: Jsonified<TransactionId>; 
  height: bigint; 
  transaction: Jsonified<Transaction>; 
}
``` 
But this led to not very convenient calls `toJson()` and `fromJson()` for `id` and `transaction` in many places in the router package. 
It is much more convenient for us to use proto message types to store in indexed-db, so I decided to use TransactionInfo in which `TransactionPersective` and `TtransactionView` will not be defined